core/lib/rom/struct.rb

```diff
@@ -33,6 +34,8 @@ module ROM
   #   end
   #
   #   class UserRepo < ROM::Repository[:users]
+  #     schema(infer: true)
+  #     auto_struct true
```

Comment:
these are settings for relations, not repos

Reply:
Ah, apologies. I made sure to test the whole script but put it in the wrong place when adding it to the comment. Fixed now